### PR TITLE
New iocage plugin for shinobi

### PR DESCRIPTION
**Pull request by allensandiego**
Created iocage plugin for shinobi and added and modified the necessary files.

- INDEX
- shinobi.json
- icons/shinobi.png

### CODE REVIEW

**Comment by fulder** [comment]
@allensandiego would you like to add a cirrus task inside the `.cirrus.yml` file for this new plugin in order to test the installation of this plugin?

File: shinobi.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "shinobi",
+  "plugin_schema": "2",
+  "release": "11.3-RELEASE",
```
Comment (sonicaj) [inline shinobi.json:4]: Can we please use `12.2-RELEASE` here ?

File: shinobi.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "packagesite": "http://pkg.FreeBSD.org/${ABI}/latest",
+  "fingerprints": {
+		  "plugin-default": [
+				  {
+				  "function": "sha256",
+				  "fingerprint": "b0170035af3acc5f3f3ae1859dc717101b4e6c1d0a794ad554928ca0cbb2f438"
+			  }
+			]
+	},
+	"official": false
```
Comment (sonicaj) [inline shinobi.json:25]: Can we fix the indentation here please ? ( and above as well )

File: shinobi.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+}
+
```
Comment (sonicaj) [inline shinobi.json:27]: Extra line

**Comment by CiaranOMara** [comment]
Shinobi is a plugin that I'd want to use. @allensandiego, would you like to revive this PR?

File: shinobi.json (hunk 1 above)
Comment (allensandiego) [reply shinobi.json]: Done. Actually changed it to 13.1-RELEASE.

File: shinobi.json (hunk 2 above)
Comment (allensandiego) [reply shinobi.json:24]: Made some fixes on the indent. Used vs-code auto format. Let me know if it's still not ok.

Comment (sonicaj) [reply shinobi.json:24]: @allensandiego please fix the indentation to use 4 spaces as indent. Thanks!

**Pull request closed without merging**